go.mod

```diff
@@ -75,3 +78,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/tgoodwin/sleeveless/client => ./custom/sleeveless-client
```

Comment:
I added our custom library code under a `./custom` directory in the repo just to facilitate `docker build`. our custom code is currently in a private repo so it's hard to pull down when the build is running from inside the docker container. In the future / when we host our custom code in a public repo, we can just pull it straight down from GitHub during the build